packages/cms-data-sync/test/run-migrations.test.ts

```diff
@@ -143,6 +144,20 @@ jest.mock('../src/research-tags/research-tags.data-migration', () => {
   };
 });
 
+var mockMigrateResearchOutputs: jest.MockedFunction<
+  typeof migrateResearchOutputs
+>;
+
+jest.mock('../src/research-outputs/research-outputs.data-migration', () => {
+  mockMigrateResearchOutputs = jest.fn().mockReturnValue({});
+  return {
+    ...jest.requireActual(
+      '../src/research-outputs/research-outputs.data-migration',
+    ),
+    migrateResearchOutputs: mockMigrateResearchOutputs,
+  };
+});
+
```

Comment:
very minor, but don't we want to check if `migrateResearchOutputs` were called or not called in the tests?

Reply:
Yes, but I'm currently writing tests for the next bit of the migration so will do it there so they don't conflict.